crates/wit-component/tests/interfaces/upstream-deps-same-name/foo.wit.print

```diff
@@ -1,7 +1,8 @@
 package foo:foo
 
 interface a {
-  use foo:a/the-name.{ty}
-  use foo:b/the-name.{ty as ty2}
+  type ty = u8
+
+  type ty2 = u8
```

Comment:
I think this may indicate a bug? (losing that these are `use` and instead generating new types)